dom.go

```diff
@@ -1612,6 +1661,14 @@ type BasicElement struct {
 	*BasicNode
 }
 
+func (e *BasicElement) Attributes() []Attribute {
+	return attributeListToAttributes(e.Get("attributes"))
+}
+
+func (e *BasicElement) Dataset() []Attribute {
+	return dataListToAttributes(e.Get("attributes"))
```

Comment:
I figured due to the fact internet explorer supports starts at  ie11. That rather than depend on the feature either breaking or working, we could use the attribute logic but perform the trimming, also seems It will need a fix to change the words into camelCase.